spec/features/bulk_screen_spec.rb

```diff
@@ -17,8 +17,8 @@
 
     find(:xpath, "//span[@class='bulk_button'][text()='Get druids from search'][not(@disabled)]").click
 
-    expect(find_button('Prepare objects')).to be_truthy
-    expect(page).to have_button('Prepare objects', disabled: false)
+    # expect(find_button('Prepare objects')).to be_truthy
+    # expect(page).to have_button('Prepare objects', disabled: false)
```

Comment:
none of the rest of this PR comments out the old way -- any reason you're doing that here?

Reply:
I've removed these now.

Reply:
github PR seems to think they are still there ...

Reply:
Okay, I *really really* did it this time.